### DISCUSSION TITLE
Plugin login error in GoLand (Jetbrains ide)

### DISCUSSION THREAD

**Discussion by Eitol** [community/community#13928] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/1043061/160925344-cab93997-b03b-4580-9cd5-bbd1eb85cc41.png)

Goland details:
Build #GO-213.7172.22, built on March 15, 2022

Licensed to Hector Oliveros
Subscription is active until April 22, 2022.

Runtime version: 11.0.14.1+1-b1751.46 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

macOS Monterey 12.2.1 (21D62)

**Comment by jansorg** [comment]
Did that happen when you launched your IDE or when you tried to sign in to GitHub Copilot?
